docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -306,53 +173,80 @@ k \frac{X^2}{2!} \frac{Y^3}{3!} \\
+```@example intro_prog_sym
+X_prod = K/(K + Z^3)
+```
+Such expressions are useful as they e.g. can designate what values to plot. Here we use this to plot $X$'s concentration and production rate across the simulation:
+```@example intro_prog_sym
+plot(sol; idxs = [X, X_prod])
+```
 
-    a. Convert to the corresponding ModelingToolkit system and then use it in
-       creating the corresponding problem.
+While the above description was brief, we hope it provide some idea of how to harness Catalyst's ability for symbolic and programmatic modelling. With it, we conclude our introduction with some advice of which part of the documentation to read next. 
 
-    b. Directly create the desired problem type from the `ReactionSystem`.
+## [Next steps](@id introduction_to_catalyst_next_steps)
+The above tutorial gives enough background to use create and simulate basic CRN models using Catalyst. The remaining documentation goes through additional features of the package. In some places, it will also provide introductions to basic theory, and general advice on how to carry out various workflows. While it would be possible to read it from start to finish, you might also select specific sections depending on your intended use of Catalyst:
+- If you have not read it already, this documentation's [home](@ref ref) page provides some useful information.
+- The [basic](@ref ref) and [advanced](@ref ref) modelling tutorials describe a range of options for model creation, and are useful to everyone who plans on using Catalyst extensively.
+- The introduction to [simulations](@ref ref), [spatial modelling](@ref ref), and [inverse problem solving](@ref ref) are useful to anyone who plans on using Catalyst for any of these purposes.
+- The [Advanced introduction to Catalyst](@ref ref) is not required to utilise any Catalyst feature. While it is not intended to be read at an early stage, it is still useful to anyone who has, or plans to, use Catalyst extensively.
+- If you are interested to learn more on how to create your models programmatically, an extensive description can be found [here](@ref programmatic_CRN_construction).
 
-   The latter is more convenient, however, the former will be more efficient if
-   one needs to repeatedly create the associated `Problem`.
-2. ModelingToolkit offers many options for optimizing the generated ODEs and
-   SDEs, including options to build functions for evaluating Jacobians and/or
-   multithreaded versions of derivative evaluation functions. See the options
-   for
-   [`ODEProblem`s](https://mtk.sciml.ai/dev/systems/ODESystem/#DiffEqBase.ODEProblem)
-   and
-   [`SDEProblem`s](https://mtk.sciml.ai/dev/systems/SDESystem/#DiffEqBase.SDEProblem).
 
 ---
-## References
+## [References](@id introduction_to_catalyst_references)
```

Comment:
Generally, I just put reference tags on all sections now, even if not used. It is not strictly required (especially this one would surprise me). However, from the previous year of working across multiple branches, it sometimes lead to annoying merge things where you modify other files when you write a tutorial. This way, if you write a new tutorial and have to reference some other section, you do not have to worry about modifying other files.